src/internal/m365/collection/drive/collection.go

```diff
@@ -282,9 +282,7 @@ func (oc *Collection) getDriveItemContent(
 		}
 
 		if clues.HasLabel(err, graph.LabelStatus(http.StatusNotFound)) || graph.IsErrDeletedInFlight(err) {
-			logger.CtxErr(ctx, err).With("skipped_reason", fault.SkipNotFound).Info("item not found")
-			errs.AddSkip(ctx, fault.FileSkip(fault.SkipNotFound, driveID, itemID, itemName, graph.ItemInfo(item)))
-
+			logger.CtxErr(ctx, err).Info("item not found, probably deleted in flight")
```

Comment:
ah, good catch, thank you.

Reply:
Actually, I take that back.  This is necessary for us to provide because at this stage in processing (item content download) we're working through the io.Reader in kopia, and are required to provide either the item data or an error.  Since we don't have item data to persist, and don't want to persist an empty blob, we have to return an error.  The only way to return an error without failing the backup is to mark that error as skippable.